app/tests/spec/views/mixins/password-strength-mixin.js

```diff
@@ -91,35 +91,13 @@ describe('views/mixins/password-strength-experiment-mixin', () => {
 
   describe('render', () => {
     beforeEach(() => {
-      sinon.stub(view, 'createExperiment');
-    });
-
-    it('designF treatment', () => {
-      sinon.stub(view, 'isInExperiment').callsFake(() => true);
-      sinon.stub(view, '_getPasswordStrengthExperimentGroup').callsFake(() => 'designF');
       sinon.stub(view, '_setupDesignF');
-
-      return view.render()
-        .then(() => {
-          assert.isTrue(view.createExperiment.calledOnceWith('passwordStrength', 'designF'));
-          assert.isTrue(view._setupDesignF.calledOnce);
-        });
     });
 
-    it('control', () => {
-      sinon.stub(view, 'isInExperiment').callsFake(() => true);
-      sinon.stub(view, '_getPasswordStrengthExperimentGroup').callsFake(() => 'control');
+    it('renders designF', () => {
       return view.render()
         .then(() => {
-          assert.isTrue(view.createExperiment.calledOnceWith('passwordStrength', 'control'));
-        });
-    });
-
-    it('not part of the experiment', () => {
-      sinon.stub(view, 'isInExperiment').callsFake(() => false);
-      return view.render()
-        .then(() => {
-          assert.isFalse(view.createExperiment.called);
+          assert.isTrue(view._setupDesignF.calledOnce);
```

Comment:
...oh, I see, that would mean extra work in the tests I guess. Still, shouldn't be *too* much effort to pull in the the other assertions to this test case, right?